lfucg-exactions/client/js/reducers/agreementsReducer.js

```diff
@@ -1,36 +1,36 @@
 import {
     GET_AGREEMENTS,
     GET_AGREEMENT_ID,
-    GET_AGREEMENT_QUERY,
     GET_ACCOUNT_AGREEMENTS,
     POST_AGREEMENT,
     PUT_AGREEMENT,
     GET_PAGINATION,
+    SEARCH_QUERY,
 } from '../constants/apiConstants';
 
 
-const agreementsReducer = (state = {}, action) => {
+const agreementsReducer = (state = [], action) => {
     const {
         endpoint,
     } = action;
     switch (endpoint) {
     case GET_AGREEMENT_ID:
     case GET_AGREEMENTS:
-    case GET_AGREEMENT_QUERY:
     case GET_ACCOUNT_AGREEMENTS:
         return action.response;
     case POST_AGREEMENT:
     case PUT_AGREEMENT:
         return {};
     case GET_PAGINATION:
+    case SEARCH_QUERY:
```

Comment:
For these reducer elements you are repeating, you could return an entire reducer (create a search reducer).  Reference Ed. Cert. uploadReducer.

Reply:
Creating a new ticket for this.